src/listeners/import-handler.ts

```diff
@@ -3,8 +3,9 @@ import { Db } from "mongodb";
 import decompress from "decompress";
 import FileType from "file-type";
 import { AssetService } from '../services/asset';
+import { TransformService } from '../services/transform';
 
-export default function importHandler(assetService: AssetService, db: Db) {
+export default function importHandler(assetService: AssetService, transformService: TransformService, db: Db, transformEnabled?: boolean) {
```

Comment:
I've always preferred default values over optional parameter? e.g. `transformEnabled: boolean = false`. Up to you if that's something that you want to change or not.